src/utils/index.js

```diff
@@ -51,3 +51,8 @@ exports.yesSayings = [
   'yeah',
   'yes',
 ];
+
+//check if the input contain letters and ends with '?' 
+exports.isAQuestion = (input) => {
+  return RegExp(/[A-z]/g).test(input) && input.endsWith('?');
+}
```

Comment:
```sh
  55:1   error  Expected space or tab after '//' in comment  spaced-comment
  55:55  error  Trailing spaces not allowed                  no-trailing-spaces
  58:2   error  Missing semicolon                            semi
```

Couple things here:
- add a space after the `//`
- remove the trailing spaces at the end of line 55
- add a semicolon `;` after the `}` on line 58

Just to be safe, once you've made all those changes, do `npm run lint` and make sure there's no errors.